src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,34 @@
+package duke.parser;
+
+import duke.command.*;
+import duke.exception.DukeException;
+
+/**
+ * Represents a parser to parse user command.
+ */
+public class Parser {
+
+    public static Command parse(String fullCommand) throws DukeException {
+        String[] description = fullCommand.trim().split(" ", 2);
```

Comment:
As a parser, I feel that this class could be further developed more to help check the format and requirements of the user input rather than pushing that role down to the Command class to handle.